cli/lib/exec/spawn.js

```diff
@@ -251,8 +251,12 @@ module.exports = {
             // the stderr logs unless we've explicitly
             // enabled the electron debug logging
             if (!debugElectron.enabled) {
```

Comment:
Since there was a PR to ignore garbage coming from Electron #4644, I'm almost warning if we can just get rid of lines 253-259 and `return true` here. Theoretically, all other stderr should either be from the plugin, or from a genuine error in Cypress.